diff --git a/src/lib_app/qml/main.qml b/src/lib_app/qml/main.qml
index d3b6823..de75a3e 100644
--- a/src/lib_app/qml/main.qml
+++ b/src/lib_app/qml/main.qml
@@ -2,7 +2,6 @@
 // See LICENSE.txt
 // https://www.219design.com
 // Software | Electrical | Mechanical | Product Design
-import QtGraphicalEffects 1.12
 import QtQuick 2.12
 import QtQuick.Controls 2.12
 import QtQuick.Controls.Universal 2.2
@@ -220,14 +219,4 @@ ApplicationWindow {
       }
     }
   }
-
-  Desaturate {
-    // turns the whole GUI application to black and white, for evaluating certain design choices.
-    visible: false // flip to true when/if you find it helpful in UI design.
-    enabled: visible
-
-    anchors.fill: main
-    source: main
-    desaturation: 1 // 0 to 1. 1 is 100% desaturated, which means grayscale
-  }
 }
